acquire/tools/decrypter.py

```diff
@@ -444,6 +461,21 @@ def main():
                     executor.shutdown(wait=True, cancel_futures=True)
                     break
 
+            results_string = textwrap.indent(
+                "\n".join(
+                    f"{file} - {'Success' if success else 'Failed'}: {message}"
+                    for file, (success, message) in sorted(results.items())
+                ),
+                " • ",
+            )
+            (progress.console.log if progress else log.info)(
+                f"\nDecrypt results (file - result: message):\n{results_string}"
+            )
+
+            if not all(successes := (success for success, _ in results.values())) or tasks:
```

Comment:
`successes` will not be what you think it is. Depending on the order of `True`'s and `False`s, it will either be `[]` or `[True]`.

Just move it out of `all()`